<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>689</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sortButton">
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="close"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="redo"/>
    <addaction name="undo"/>
    <addaction name="actionclear"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="draw_a_line"/>
    <addaction name="draw_a_rectangle"/>
    <addaction name="draw_a_circle"/>
   </widget>
   <widget class="QMenu" name="menucolor">
    <property name="title">
     <string>color</string>
    </property>
    <addaction name="color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDraw"/>
   <addaction name="menucolor"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="redo"/>
   <addaction name="undo"/>
   <addaction name="color"/>
   <addaction name="draw_a_circle"/>
   <addaction name="draw_a_rectangle"/>
   <addaction name="draw_a_line"/>
   <addaction name="actionclear"/>
   <addaction name="about"/>
   <addaction name="close"/>
  </widget>
  <action name="close">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/exit icon.png</normaloff>:/icons/exit icon.png</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="redo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/redo icon.png</normaloff>:/icons/redo icon.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
  </action>
  <action name="undo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/undo icon.png</normaloff>:/icons/undo icon.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="about">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/about icon.png</normaloff>:/icons/about icon.png</iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="draw_a_line">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/line icon.png</normaloff>:/icons/line icon.png</iconset>
   </property>
   <property name="text">
    <string>draw a line</string>
   </property>
  </action>
  <action name="draw_a_rectangle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/rectangle icon.png</normaloff>:/icons/rectangle icon.png</iconset>
   </property>
   <property name="text">
    <string>draw a rectangle </string>
   </property>
  </action>
  <action name="draw_a_circle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/cricle icons.png</normaloff>:/icons/cricle icons.png</iconset>
   </property>
   <property name="text">
    <string>draw a circle</string>
   </property>
  </action>
  <action name="color">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/color icon.png</normaloff>:/icons/color icon.png</iconset>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/clear-icon-png-19.jpg</normaloff>:/icons/clear-icon-png-19.jpg</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
